JerryCheong.Calculator/Program.cs

```diff
@@ -0,0 +1,263 @@
+﻿using System.Text.RegularExpressions;
+using CalculatorLibrary;
+
+namespace CalculatorProgram
+{
+    class Program
+    {
+        private Calculator calculator;
+        public Program()
+        {
+            calculator = new Calculator();
+        }
+
+        static void Main(string[] args)
+        {
+            Program program = new Program();
+            // Display title as the C# console calculator app.
+            Console.WriteLine("Console Calculator in C#\r");
+            Console.WriteLine("------------------------\n");
+
+            // Declare variables and set to empty.
+            // Use Nullable types (with ?) to match type of System.Console.ReadLine
+            string? numInput1 = "";
+            string? numInput2 = "";
+
+            bool endApp = false;
+            while (!endApp)
+            {
+                // Ask the user to type the first number.
+                Console.Write("Type a number, and then press Enter: ");
+                numInput1 = Console.ReadLine();
+
+                double cleanNum1 = 0;
+                while (!double.TryParse(numInput1, out cleanNum1))
+                {
+                    Console.Write("This is not valid input. Please enter a numeric value: ");
+                    numInput1 = Console.ReadLine();
+                }
+
+                // Ask the user to type the second number.
+                Console.WriteLine("Type another number, and then press Enter: ");
+                Console.WriteLine("OR");
+                Console.WriteLine("Choose an operator from the following list:");
+                Console.WriteLine("\tr - SquareRoot");
+                Console.WriteLine("\tt - TenPower");
+                Console.WriteLine("\tsine - Sin");
+                Console.WriteLine("\tcosine - Cos");
+                Console.WriteLine("\ttangent - Tan");
+
+                numInput2 = Console.ReadLine();
+
+                double cleanNum2 = 0;
+                while (string.IsNullOrEmpty(numInput2))
+                {
+                    Console.Write("Input cannot be empty. Please enter a numeric value or choose an operator:");
+                    numInput2 = Console.ReadLine();
+                }
+                while (!double.TryParse(numInput2, out cleanNum2) && !Regex.IsMatch(numInput2, "[r|t|sine|cosine|tangent]"))
```

Comment:
if numInput2 = 's' is entered the program didn't act accordingly.
There is also a bug when you catch an exception in your CalculatorLibrary.
I will suggest that you should entered different invalid inputs for the second input.
Please fix this issue.

Reply:
I have fixed the issue, please have a look again. Thank you!